lab_1_classify_by_unigrams/main.py

```diff
@@ -49,6 +77,15 @@ def create_language_profile(language: str, text: str) -> dict[str, str | dict[st
 
     In case of corrupt input arguments, None is returned
     """
+    if (language is not None and isinstance(language, str)
```

Comment:
return None

Reply:
I have "return None" at the end of the cycle. I tried to place it in a different way, but got mistakes with returns (there was something like  "you cannot return value and None").

Reply:
None returns if, for example, language isn't string

Reply:
done